### DISCUSSION TITLE
AeroVal - aerosol diagnostics tool

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#32] [Oslo-Aero]
We will use AeroVal for the evaluation of aerosols in NorESM2.5. Let's discuss challenges, missing variables, missing observations, and other questions in this thread :)

**Comment by andrearosendahl** [comment]
For now I have included AOD and SO4 on Aeroval, but the AOD from the noresm2.5 models are only AODVIS and it would be nice to have DOD550 as well. Can that be added for the next run @mvertens ? 
Please let me know if there are other variables you would want me to look at.
Adding the Aeroval website with the preliminary results: https://aeroval-test.met.no/andrear/pages/intercomp/?project=noresm&experiment=cases-vs-cmip&parameter=od550aer&station=ALL

**Comment by MichaelSchulzMETNO** [comment]
Another wish - would be to have a present day simulations, with PD emissions, thus the end of the historical period. 2001-2014 for instance, or why not until 2020. No spinup of climate needed for inspecting the aerosol fields. @mvertens

**Comment by mvertens** [comment]
@MichaelSchulzMETNO @andrearosendahl - I am currently working on getting a code base in place both for the workshop next week and in order to distribute carrying out the simulations. Once the code base is out we can decide who will carry out the new needed simulations.

**Comment by MichaelSchulzMETNO** [comment]
Other observable variables which could be plotted:

**Total AOD** versus MODIS AOD 

**Coarse mode AOD** = Dust AOD   aeroval cong:   "obs_id": "AeronetSDAV3Lev2.daily", "obs_vars": ["od550dust"] , NorESM variable : D550_DU

**WetSO4** "obs_id": "EBASMC", NoreSM: "'SO4_A1SFWET+SO4_A2SFWET+SO4_ACSFWET+SO4_NASFWET+SO4_PRSFWET+SO4_A1_OCWSFWET+SO4_A2_OCWSFWET+SO4_AC_OCWSFWET+SO4_NA_OCWSFWET+SO4_PR_OCWSFWET'

variable list mapping for NorESM to cmor is here: 
https://github.com/NorESMhub/noresm2cmor/blob/master/namelists/var_CMIP6_NorESM2_default.nml

**Reply by andrearosendahl** [reply]
Total AOD vs MODIS AOD is already plotted.

**Reply by MichaelSchulzMETNO** [reply]
yes - I missed that one.

**Comment by oyvindseland** [comment]
Does AOD_VIS from the model take into account that for the default set-up AOD_VIS is calculated only at daytime.

**Comment by Ovewh** [comment]
From the aeroval evaluations I did previously i also included the angstrom exponent and finemode AOD from Aeronet.

**Comment by oyvindseland** [comment]
NorESM-hybrid22 seems to be an outlier compared to the other NorESM2.5 simulations. Is this a simulation with corrected dust emissions?

**Reply by MichaelSchulzMETNO** [reply]
The outliers are the other simulations :) Exactly - corrected emissions of dust (and seasalt?) in NorESM-hybrid22

**Reply by mvertens** [reply]
I've updated the experiment description to make this clearer.
